Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG commitHash=9539a1b36d3fba12cbb54119c4f5209e10371315
```

Comment:
@joostjager How do you feel about the commitHash in the Dockerfile? It would need to be updated for each new "release" or version of the image you want to push/publish.

Reply:
Actually, I think that it would be better to just ADD the local directory and build from that? Updating the commit hash everytime isn't ideal. And not specifying isn't great either, because you need to build with no cache to force the new version to be picked up.

The dockerfile on https://github.com/lightningequipment/circuitbreaker/pull/27/files is using that approach currently.

Reply:
Yeah, that makes sense

Reply:
@joostjager fixed 🚀